torchgeo/models/farseg.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Foreground-Aware Relation Network (FarSeg) implementations."""
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision.models import resnet
+from torchvision.ops import FeaturePyramidNetwork as FPN
+from typing import List, Tuple
+import math
+from collections import OrderedDict
+
+
+class FarSeg(nn.Module):
+    """Foreground-Aware Relation Network (FarSeg).
+
+    'Foreground-Aware Relation Network for Geospatial Object Segmentation
+    in High Spatial Resolution Remote Sensing Imagery', Zheng et al. (2020)
```

Comment:
It might be better to describe what the model actually does instead of the title of the paper, since the link to the paper is already below.